xpdtools/pipelines/qoi.py

```diff
@@ -21,6 +21,13 @@ def max_gr_mean(pdf, **kwargs):
     return locals()
 
 
+def fluc_high_q(iq_comp, high_q_val=45, **kwargs):
+    q = iq_comp.pluck(0)
+    iq = iq_comp.pluck(1)
+    high_q_fluc = avg_curvature(iq, q, high_q_val)
```

Comment:
You might use the `q` and `mean` and combine them, see: https://github.com/xpdAcq/xpdtools/blob/master/xpdtools/pipelines/raw_pipeline.py#L340
This needs to run through a starmap.